crates/store/src/db/sql/mod.rs

```diff
@@ -350,8 +360,7 @@ pub fn select_account_delta(
     ])?;
     while let Some(row) = rows.next()? {
         let vault_key_data = row.get_ref(1)?.as_blob()?;
-        let vault_key = Word::read_from_bytes(vault_key_data)?;
-        let asset = NonFungibleAsset::try_from(vault_key)
+        let asset = NonFungibleAsset::read_from_bytes(vault_key_data)
             .map_err(|err| DatabaseError::DataCorrupted(err.to_string()))?;
```

Comment:
This is a separate issue but while testing I realized that we were using the `account_non_fungible_asset_updates::vault_key` to store a serialized word that is used to build the non fungible asset.
Before the ID changes, the vault key and the asset word were the same but this is no longer the case. Maybe we should change the name